src/Admin/BaseFieldDescription.php

```diff
@@ -132,6 +132,23 @@ abstract class BaseFieldDescription implements FieldDescriptionInterface
      */
     private static $fieldGetters = [];
 
+    /**
+     * NEXT_MAJOR: Remove the null default value and restrict param type to `string`.
+     */
+    public function __construct(?string $name = null)
```

Comment:
is the persistence bundle the one in charge for creating those?

Reply:
Yes, why ?

I plan to do a PR on them after a SonataAdmin release.

Reply:
Just to know that this change can be applied there.